Source/WebCore/layout/formattingContexts/inline/InlineItemsBuilder.cpp

```diff
@@ -450,21 +449,19 @@ static inline void buildBidiParagraph(const RenderStyle& rootStyle, const Inline
     for (size_t index = 0; index < inlineItemList.size(); ++index) {
         auto& inlineItem = inlineItemList[index];
         auto& layoutBox = inlineItem.layoutBox();
-        auto mayAppendTextContentAsOneEntry = is<InlineTextBox>(layoutBox) && !TextUtil::shouldPreserveNewline(downcast<InlineTextBox>(layoutBox));
```

Comment:
this variable has commentary purpose.

Reply:
The one downside is that I'll have to declare two variables then before the if statement putting them in a somewhat wider scope, but I guess that was kinda the case already.